consensus/poa_validator.go

```diff
@@ -105,15 +111,15 @@ func (c *Consensus) authorityCacheHandler(candidates *poa.Candidates, header *bl
 			for _, r := range receipts {
 				for _, o := range r.Outputs {
 					for _, ev := range o.Events {
-						if header.Number() >= c.forkConfig.HAYABUSA && ev.Address == builtin.Staker.Address {
+						if header.Number() >= p.forkConfig.HAYABUSA && ev.Address == builtin.Staker.Address {
```

Comment:
Can you add a comment here?
// after HAYABUSA, authorities are allowed to migrate to staker contract, so any staker contract event(AddValidation, StakeIncreased/Decreased/Withdrawn) will need to invalidate cache